### PR TITLE
Action Item: Dick, Charlie, and Henk improve wording in first use case

### PR DESCRIPTION
**Pull request by henkbirkholz**
Signed-off-by: Henk Birkholz <henk.birkholz@sit.fraunhofer.de>

### CODE REVIEW

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM! I am in the process of reviewing the entire document!

File: draft-birkholz-scitt-software-use-cases.md [hunk 1]
```diff
@@ -85,21 +85,21 @@ deployment chain diagram here
 
 # Software Supply Chain Use Cases
 
-## Trust Bond between Package Supplier and the Signing Authority
+## Trustworthiness Relationship between Product Supplier and the Signer of the Product
```
Comment (joshuagl) [inline draft-birkholz-scitt-software-use-cases.md]:
```suggestion
## Trustworthiness Relationship between Product Supplier and the Distributor of the Product
```

File: draft-birkholz-scitt-software-use-cases.md [hunk 2]
```diff
@@ -85,21 +85,21 @@ deployment chain diagram here
...
 
 A certain software component product is created and packaged by a Supplier.
 The package by itself does not include a proof of authenticity.
 A signing authority is tasked with adding a proof of authenticity.
-Trust has to be established from the Supplier towards the Signing Authority - and vice versa.
-The mutual trust relationship (trust bond) between Supplier and Signing Authority is established per each individual software component package.
+Trustworthiness has to be established from the Supplier towards the Signing Authority - and vice versa.
```
Comment (joshuagl) [inline draft-birkholz-scitt-software-use-cases.md]:
```suggestion
Trustworthiness has to be established from the Supplier towards the Distributor - and vice versa.
```

File: draft-birkholz-scitt-software-use-cases.md [hunk 3]
```diff
@@ -85,21 +85,21 @@ deployment chain diagram here
...
+The mutual trustworthiness relationship between Supplier and Signing Authority is established per each individual software component package.
```
Comment (joshuagl) [inline draft-birkholz-scitt-software-use-cases.md]:
```suggestion
The mutual trustworthiness relationship between Supplier and Distributor is established per each individual software component package.
```

File: draft-birkholz-scitt-software-use-cases.md [hunk 4]
```diff
@@ -85,21 +85,21 @@ deployment chain diagram here
...
 
 A consumer of a released software wants:
 
-* to understand and verify that an actual trust bond exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
+* to understand and verify that an actual trustworthiness relationship exists between the Supplier of a certain software component package and the Signing Authority of that software component package.
```
Comment (joshuagl) [inline draft-birkholz-scitt-software-use-cases.md]:
```suggestion
* to understand and verify that an actual trustworthiness relationship exists between the Supplier of a certain software component package and the Distributor of that software component package.
```

**Review by joshuagl — COMMENTED** [review]
From discussion with Henk I believe the use-case here is an app store-like promotion scenario. If that's true, I recommend we refer to this as the relationship between a supplier/producer and the distributor. While signing is a property of the relationship between an app creator and an app store we can increase the clarity of the use-case by describing the roles in the relationship (producer/distributor/consumer).

I've made some suggestions on the PR to replace "Signing Authority" with "Distributor".

**Review by henkbirkholz — COMMENTED** [review]
In general, I am okay to relax the scope a bit and move from Signing Authority to Distributor.

Anyone opposing this change? We will bring in up in the next meeting on 2023-01-23

**Comment by rjb4standards** [comment]
Henk,

 

The Distributor isn’t always the signing authority. It’s true in some app stores, but beyond that the model breaks, down. Walmart doesn’t digitally sign software it sells/distributes.

 

Thanks,

 

Dick Brooks

  

Active Member of the CISA Critical Manufacturing Sector, 

Sector Coordinating Council – A Public-Private Partnership

 

 <https://reliableenergyanalytics.com/products> Never trust software, always verify and report! ™

 <http://www.reliableenergyanalytics.com/> http://www.reliableenergyanalytics.com

Email:  ***@***.***> ***@***.***

Tel: +1 978-696-1788

**Comment by henkbirkholz** [comment]
Then this is going on the agenda for Monday, too. Thanks Dick

**Comment by rjb4standards** [comment]
Thanks, Henk.

 

Thanks,

 

Dick Brooks

  

Active Member of the CISA Critical Manufacturing Sector, 

Sector Coordinating Council – A Public-Private Partnership

 

 <https://reliableenergyanalytics.com/products> Never trust software, always verify and report! ™

 <http://www.reliableenergyanalytics.com/> http://www.reliableenergyanalytics.com

Email:  ***@***.***> ***@***.***

Tel: +1 978-696-1788

File: draft-birkholz-scitt-software-use-cases.md (hunk 4 above)
Comment (yogeshbdeshpande) [reply draft-birkholz-scitt-software-use-cases.md]: Accepting this for now however effectively we have 3 roles , 
* Supplier
* Signer 
* Distributor

**Pull request merged**